Frontend/ui-driver/src/Flow.purs

```diff
@@ -2078,6 +2078,9 @@ homeScreenFlow = do
     GO_TO_RIDE_DETAILS_SCREEN -> do 
       tripDetailsScreenFlow
     POST_RIDE_FEEDBACK state-> do 
+      when (state.data.endRideData.rating == 5) $ do
```

Comment:
`when (state.data.endRideData.rating == 5) $ pure $ JB.launchInAppRatingPopup unit`

Just this will be fine no need of extra pure unit.

Reply:
It's a Effect Unit function, tried doing that it gave error

Reply:
okay let me check